ui-tests/src/test/java/org/openmrs/reference/page/MarkPatientDeceasedPage.java

```diff
@@ -0,0 +1,116 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+
+public class MarkPatientDeceasedPage extends Page {
+
+    private static final String URL = "/coreapps/markPatientDead.page";
+    private static final By DICTIONARY_LINK= By.id("dictionaryNavLink");
+    private static final By ADD_NEW_CONCEPT = By.linkText("Add new Concept");
+    private static final By CONCEPT_NAME = By.id("namesByLocale[en].name");
+    private static final By ADD_ANSWER_BUTTON = By.cssSelector("#codedDatatypeRow > td > table > tbody > tr > td.buttons > input:nth-child(1)");
+    private static final By CAUSE_BOX = By.id("newAnswerConcept");
+    private static final By FEVER = By.xpath("/html/body/ul[1]/li[2]/a");
+    private static final By ADD_BUTTON = By.xpath("/html/body/div[5]/div[11]/button[1]");
+    private static final By SAVE_CONCEPT_BUTTON = By.xpath("/html/body/div[1]/div[3]/div[2]/form/div/input[1]");
+    private static final By SEARCH_CONCEPT_FIELD = By.id("inputNode");
+    private static final By FIRST_RESULT = By.xpath("//*[@id=\"openmrsSearchTable\"]/tbody/tr[1]/td/span");
+    private static final By CONCEPT_ID = By.xpath("//*[@id=\"conceptTable\"]/tbody/tr[1]/td");
+    private static final By ADMINISTRATION_PAGE = By.id("administrationNavLink");
+    private static final By ADVANCED_SETTINGS = By.linkText("Advanced Settings");
+    private static final By CONCEPT_VALUE_FIELD = By.xpath("//*[@id=\"globalPropsList\"]/tr[133]/td[2]/input");
+    private static final By SAVE_CONCEPT_VALUE = By.xpath("//*[@id=\"buttonsAtBottom\"]/input[1]");
+    private static final By HOME_LINK = By.id("homeNavLink");
+    private static final By MARK_PATIENT_DECEASED = By.id("org.openmrs.module.coreapps.markPatientDead");
+    private static final By CHECK_BOX = By.id("deceased");
```

Comment:
Use a better element readable name rather than CHECK_BOX. 

Hint: CHECK_BOX for what ?